editor/src/components/canvas/canvas-strategies/escape-hatch-strategy.spec.tsx

```diff
@@ -137,16 +137,21 @@ const mixedPinsMetadata = {
   } as ElementInstanceMetadata,
 }
 
-function dragBy15Pixels(
+function dragBy15Pixels(editorState: EditorState, metadata: ElementInstanceMetadataMap) {
```

Comment:
`dragBy21.2132Pixels`

Reply:
kidding aside, this was a significant readability improvement for these tests